### ISSUE TITLE
Ship compiler as a native Linux executable

### ISSUE THREAD

**Issue by abitrolly** [kaitai-io/kaitai_struct#812]
As a DevOps I find convenience in Go tools that often ship as a standalone Linux executables that don't require `sudo` installation. I would prefer https://kaitai.io/#download to provide an option for Linux that doesn't require installing packages or Java runtime.

**Comment by GreyCat** [comment]
Issue's title suggests that it's a duplicate of https://github.com/kaitai-io/kaitai_struct/issues/519, but inside you claim what you're really looking is not a "native Linux executable", but a everything-bundled no-install packet?

**Comment by abitrolly** [comment]
@GreyCat yes, I would like no-install option for https://github.com/abitrolly/ksykaitai Not sure about everything-bundled if compiler doesn't need any additional files to compile my `.ksy`.

**Comment by dgelessus** [comment]
If you just care about running ksc without `sudo` installation and don't need a single self-contained binary, you can download the universal (jar) version of ksc and a standalone prebuilt Linux JRE (such as [AdoptOpenJDK](https://adoptopenjdk.net/releases.html)). Then you can use the JRE (without installation) to run ksc using a command like `JAVA_HOME=/path/to/downloaded/jre /path/to/downloaded/ksc/bin/kaitai-struct-compiler <args>`.
